nengo_viz/static/viz_slider.js

```diff
@@ -151,6 +128,35 @@ VIZ.Slider = function(args) {
 VIZ.Slider.prototype = Object.create(VIZ.Component.prototype);
 VIZ.Slider.prototype.constructor = VIZ.Slider;
 
+VIZ.Slider.prototype.set_value = function(slider_index, value) {
+    //Get the slider
+    var target = this.sliders[slider_index].div;
+
+    //important for 2d sliders
+    var x_pos = target.getAttribute('fixed-x'); 
+    
+    //Get the scaled value
+    var point = this.scale(value);
+
+    //Get the slider height
+    var height = parseInt(target.style.height);
```

Comment:
Hmm, I hadn't realized we aren't storing ```args.height``` anywhere, so we have to get it through something ugly like this.  I'm pretty sure we should store ```args.height``` and ```args.width``` so that we can just access them via ```this.width``` and ```this.height```.  But that can be a separate PR.